prospero-common/src/main/java/org/wildfly/prospero/api/ProvisioningDefinition.java

```diff
@@ -51,47 +54,73 @@ public class ProvisioningDefinition {
 
     public static final RepositoryPolicy DEFAULT_REPOSITORY_POLICY = new RepositoryPolicy(true, RepositoryPolicy.UPDATE_POLICY_ALWAYS, RepositoryPolicy.CHECKSUM_POLICY_FAIL);
 
+    /**
+     * Galleon feature pack location. Can be either a well-known name (like "eap-8.0") that references a predefined combination
+     * of Galleon configuration and channels, or a standard Galleon feature pack location string (see
+     * https://docs.wildfly.org/galleon/#_feature_pack_location).
+     */
     private final String fpl;
+
+    /**
+     * This field would only contain channels parsed from the prosper-known-combinations.yaml file ("well-known feature packs"),
```

Comment:
The comments talk about command line attributes, while the `ProvisioningDefinition` is in prospero-common. I think the ProvisioningDefinition should be moved to cli module

Reply:
I will just change the attribute references to builder methods. I think this class is useful as part of programmatic API in common.